altitude_parsing.c

```diff
@@ -0,0 +1,77 @@
+#include "altitude_parsing.h"
+
+static int32_t altitude = -999;
+static bool new_altitude = false;
+static char string[16];
+
+char rx_pool[32]; //32 bytes should be plenty
+
+static srb_ctx_t rx_buf;
+
+int32_t get_altitude(){
```

Comment:
Best practice in C is to declare functions taking no arguments as taking `void` as an argument, i.e. `int32_t get_altitude(void)`

One of the quirks in C is that functions declared without that can take any number of arguments.